rotkehlchen/chain/ethereum/modules/uniswap/v3/decoder.py

```diff
@@ -89,6 +89,14 @@ def _find_from_asset_and_amount(events: list['EvmEvent']) -> tuple[Asset, FVal]
     return from_asset, from_amount
 
 
+def _asset_is_eth(event_asset: 'Asset', pool_token: 'Asset') -> bool:
+    """Compare two assets evaluating as equal WETH and ETH"""
```

Comment:
From the current name and description I don't understand what this did. Also isn't pool token always an EVM token?

```suggestion
def _compare_with_maybe_eth_token(asset: 'Asset', pool_token: 'EVMToken') -> bool:
    """Compares an asset with a pool token, considering ETH equal to WETH"""
```